src/block/block_ckpt.c

```diff
@@ -653,17 +653,77 @@ __ckpt_process(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_CKPT *ckptbase)
     return (ret);
 }
 
+/*
+ * __ckpt_add_blkmod_entry --
+ *     Add an offset/length entry to the bitstring based on granularity.
+ */
+static int
+__ckpt_add_blkmod_entry(
+  WT_SESSION_IMPL *session, WT_BLOCK_MODS *blk_mod, wt_off_t offset, wt_off_t len)
```

Comment:
`len` isn't a `wt_off_t`, I don't think, it's a `uint32_t` in `block.h`.

Reply:
You're right it is. Not sure what I was looking at yesterday when I would've sworn I saw it declared a `wt_off_t`. Fixed.

Reply:
Looking again, `len` *is* a `wt_off_t`. From block.h:
```
struct __wt_ext {
    wt_off_t off;  /* Extent's file offset */
    wt_off_t size; /* Extent's Size */
```